packages/vue/src/use-chat.ui.test.tsx

```diff
@@ -136,7 +136,11 @@ describe('data protocol stream', () => {
   });
 
   it('should show error response', async () => {
-    mockFetchError({ statusCode: 404, errorMessage: 'Not found' });
```

Comment:
are imports changing? need organize?

Reply:
yep they changed, is there something specific you want to address?